staging/src/k8s.io/cloud-provider/volume/helpers/zones_test.go

```diff
@@ -898,7 +898,7 @@ func TestSelectZonesForVolume(t *testing.T) {
 			Reject: true,
 		},
 
-		// Key specified in AllowedTopologies is not LabelFailureDomainBetaZone [Fail]
+		// Key specified in AllowedTopologies is not LabelTopologyZone [Fail]
```

Comment:
Can you keep one test case with AllowedTopologies using the beta label?

Reply:
Done. Add two test cases one for node using beta label one for AllowedTopologies using beta label